app/views/shared/administrate_batch_actions/_button.html.erb

```diff
@@ -1 +1 @@
-<%= link_to name, path, class: html_options[:class], data: { batch_action_option: 'button', confirm: html_options[:confirm] }, method: :post %>
+<%= link_to name, path, class: "btn disabled #{html_options[:class]}", data: { batch_action_option: 'button', confirm: html_options[:confirm] }, method: :post %>
```

Comment:
Should we be merging `html_options[:class]` with the default classes? That way if someone does not want to use the `btn` class they can overwrite it?

Reply:
Actually, the `btn disabled` works together to make the button as disabled, if it's a link, it sounds tricky to be a disabled link, right?